HEC.FDA.Model/Spatial/Extensions/Utility.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Utility.Memory;
+
+namespace HEC.FDA.Model.Spatial.Extensions;
+public static class Utility
+{
+    /// <summary>
+    /// Tries to get the value of the specified column as the specified type. If the type is wrong, or column doesn't exist, returns fallback value
+    /// </summary>
+    public static T TryGetValueAs<T>(this TableRow row, string columnName, T fallbackValue = default) {
+		try
+		{
+			if(string.IsNullOrWhiteSpace(columnName))
```

Comment:
ValueAs throws if we ask for a column that doesn't exist. This stops that throw when we check for mapped attributes that weren't set (come in as empty strings for column names). Throwing exceptions is super expensive though, so we check full null or whitespace and gracefully "fail" first.  The catch shouldn't ever hit without something exceptional happening, like someone modifying the column names of the shapefile we're using after import.